packages/fxa-auth-server/lib/senders/emails/templates/subscriptionSubsequentInvoice/en.ftl

```diff
@@ -0,0 +1,24 @@
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionSubsequentInvoice-subject = { $productName } payment received
+subscriptionSubsequentInvoice-title = Thank you for being a subscriber!
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionSubsequentInvoice-content-received = We received your latest payment for { $productName }.
+subscriptionSubsequentInvoice-content-invoice-number-label = Invoice Number:
+# Variables:
+#  $invoiceNumber (String) - The invoice number of the subscription invoice, e.g. 8675309
+subscriptionSubsequentInvoice-content-invoice-number-value = { $invoiceNumber }
```

Comment:
I'm kind of curious how localizers see this with it just being a number. I had a similar ftl ID for only `productName` and was able to remove that since the product names don't get localized, but numbers should so... let's leave for now, but curious what @peiying2 would say.

Reply:
You can remove this too, as it is a fixed but unique number and shouldn't be touched. Thanks for checking in!